src/utils/index.js

```diff
@@ -0,0 +1,9 @@
+export const findById = async (Model, id) => {
+  const result = await Model.findOne({ where: { id } });
+  return result;
+};
+
+export const findByEmail = async (Model, email) => {
```

Comment:
This is excessive and not needed since we have a relationship between article and users table. The database should throw an error for on existent user trying to update its resource.